src/components/badge/badge.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Badge from "./badge.component";
+
+const renderComponent = (props = {}) =>
+  render(<Badge {...props}>Button</Badge>);
+
+describe("Badge", () => {
+  it("should render correctly if counter is a number", async () => {
```

Comment:
Also most of these don't need to be `async` tests as we're using synchronous `getBy` queries instead of `findBy` which is asynchronous and needs awaiting

Reply:
Yeah I'll remove both of those tests, they don't seem to affect coverage